sapp/ui/frontend/src/Source.js

```diff
@@ -32,6 +33,18 @@ type Range = $ReadOnly<{
   to: Location,
 }>;
 
+const modes = {
+  py: "text/x-python",
```

Comment:
Since there are constant values here that are reused (e.g. `"text/x-python"`), should we create a Enum to store these mime types instead?

But since that might be overkill, I wonder if anyone else has opinions on this

Reply:
I could move that to an enum for sure.